packages/headless/src/api/search/search-api-client.ts

```diff
@@ -74,7 +75,13 @@ export class SearchAPIClient implements FacetSearchAPIClient {
     req: PlanRequest
```

Comment:
Some requests may use `${orgId}.admin.org.coveo.com`, which I think is the default for anything that's not Search API nor Analytics related. I recommend double-checking that we have at-least one functional (or e2e) test that sends a query per request type.

Reply:
Which requests calls `.admin.` ?
I'm curious

Reply:
I recall that anything related to Insights is under `.admin.`

Reply:
No, insights panel is under `/search`.

Insight panel is working correctly with these changes.